application/cms/views.py

```diff
@@ -1253,6 +1254,11 @@ def set_measure_order():
             pages = MeasureVersion.query.filter_by(guid=p["guid"], parent_guid=p["subtopic"]).all()
             for page in pages:
                 page.position = p["position"]
+
+            if pages:
```

Comment:
Is it worth putting "Deprecated" or "Remove me" comments on the old code in all the places where we're duplicating stuff here?  I'm not saying definitely do this.  But maybe?

Reply:
I'm not super worried about that - I think it will be obvious when we get to pulling out the old code? (i.e. when we remove the position field, this thing will totally break, so it can't be missed)